quickstart/101-front-door-classic/providers.tf

```diff
@@ -0,0 +1,19 @@
+# Configure the Azure provider
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 3.27.0"
+    }
+
+    random = {
+      source = "hashicorp/random"
+    }
```

Comment:
Please include the random provider version here.

Reply:
Updated.